.github/workflows/build-and-publish.yml

```
@@ -2,10 +2,12 @@ name: Build and Publish
 
 on:
   push:
+    branches: [ 'develop' ]
+    tags: [ 'v*.*.*' ]
```

If you want to restrict tag format, you need to use a regular expression like this:
```suggestion
    tags: [ 'v[0-9]+.[0-9]+.[0-9]+' ]
```
But better not to restrict it for now, I think.

---

Hmm, I'm not 100% sure it fully supports regular expressions. Maybe just the character set works. Here are examples: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

---

Yeah, if it did the `.` characters would have to be escaped. Oh well, like I said, better to just keep it simple for now.